ext/dom/xpath.c

```diff
@@ -446,6 +446,60 @@ PHP_METHOD(DOMXPath, registerPhpFunctionNS)
 	);
 }
 
+/* {{{ */
+PHP_METHOD(DOMXPath, quote) {
+	char *input;
+	size_t input_len;
+	smart_str output = {0};
+
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "s", &input, &input_len) ==
```

Comment:
This is a style nit that is not mandatory to follow, but I find that the 80 column wrapping rule makes reading harder. I personally use 120 columns.

Reply:
switch to a 13" laptop and read it again 😄 hopefully we'll get some automated formatting going soon